metron-interface/scripts/run-all-ui-test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+current_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd "$current_path/.."
+
+echo ""
+echo "      _/_/    _/                        _/                    _/    _/  _/_/_/   "
+echo "   _/    _/  _/    _/_/    _/  _/_/  _/_/_/_/    _/_/_/      _/    _/    _/      "
+echo "  _/_/_/_/  _/  _/_/_/_/  _/_/        _/      _/_/          _/    _/    _/       "
+echo " _/    _/  _/  _/        _/          _/          _/_/      _/    _/    _/        "
+echo "_/    _/  _/    _/_/_/  _/            _/_/  _/_/_/          _/_/    _/_/_/       "
+echo ""
+
+./metron-alerts/scripts/run-all-test.sh
+
+echo ""
+echo "     _/_/_/                          _/_/  _/                _/    _/  _/_/_/   "
+echo "  _/          _/_/    _/_/_/      _/            _/_/_/      _/    _/    _/      "
+echo " _/        _/    _/  _/    _/  _/_/_/_/  _/  _/    _/      _/    _/    _/       "
+echo "_/        _/    _/  _/    _/    _/      _/  _/    _/      _/    _/    _/        "
+echo " _/_/_/    _/_/    _/    _/    _/      _/    _/_/_/        _/_/    _/_/_/       "
+echo "                                                _/                              "
+echo "                                           _/_/                                 "
+echo ""
+
+./metron-config/scripts/run-all-test.sh
```

Comment:
It is hard to distinguish which logs belong to which UI so I added this to the shell script.